modular_citadel/code/game/jobs/job/security_cit.dm

```diff
@@ -0,0 +1,4 @@
+// Per request
+
+/datum/job/officer
+	alt_titles = list("Brig Officer")
```

Comment:
This will override the "Junior officer" alt-title. Readd the alt-title that was overridden